sdk/main/include/MnemonicBIP39.h

```diff
@@ -73,6 +79,27 @@
    */
   static MnemonicBIP39 generate24WordBIP39Mnemonic();
 
+  /**
+   * Generate an ED25519PrivateKey from this Mnemonic using the input passphrase at the specified index.
```

Comment:
This doc should note that index must be unhardened

Reply:
Fixed